### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by Gander7**
I updated the readme and added contribute.md.
Read them over and let me know what you think, i'm not attached to any of it.
I added a tentative possible todos section in contribute.md that could start discussion on next step.

### CODE REVIEW

File: CONTRIBUTE.md [hunk 1]
```diff
@@ -1 +1,75 @@
...
+* Define Coding Standards
+* Create a wiki/place ot get help
+* Create online demo or how to use
+* Possible create/define API?
+
+## Code of Conduct
+
+### Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
```
Comment (plusjamal) [inline CONTRIBUTE.md]: I dont' know if I can agree to these terms, zach.  You're asking a lot of me here...:)

File: CONTRIBUTE.md [hunk 2]
```diff
@@ -1 +1,75 @@
...
+
+### Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at [INSERT EMAIL ADDRESS]. All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good faith may face temporary or permanent repercussions as determined by other members of the project's leadership.
+
+### Attribution
+
+This Code of Conduct is adapted from the Contributor Covenant, version 1.4, available [here](http://contributor-covenant.org/version/1/4).
```
Comment (plusjamal) [inline CONTRIBUTE.md]: So many little things I'm learning about.  Book marked this link.

File: CONTRIBUTE.md (hunk 1 above)
Comment (Gander7) [inline CONTRIBUTE.md]: to be honest, didn't even read it myself lol

File: CONTRIBUTE.md [hunk 3]
```diff
@@ -1 +1,75 @@
...
+5. All code should be commented.
+6. Ensure any install, build, or environment dependencies are removed before the pull request is submitted. 
+7. Increase version numbers, as needed, that this pull request would represent. See [README.md#Versioning](../README.md)
+8. You may merge the pull request once you have the sign-off of at least one of the owners, or if you do not have permission, you may request an owner/reviewer to merge it for you.
+
+## Possible TODOs
+Looking to help out but don't know what to do? 
+The following is up for discussion, contact us about helping out with the following:
+
+* Implement MongoDB
```
Comment (Gander7) [inline CONTRIBUTE.md]: definitely, that's what I was thinking as well

File: README.md [hunk 4]
```diff
@@ -1,17 +1,36 @@
...
-Please read CONTRIBUTING.md for details on the process of contributing.
+Please read [CONTRIBUTING.md](../CONTRIBUTE.md) for details on the process of contributing a bug report or a new feature request.
 
 ## Versioning
-We use SemVer for versioning.
+We use [SemVer](http://semver.org/) for versioning.
 
-## Authors
+## Owners
 - Jamal Rahali
```
Comment (Gander7) [inline README.md]: DARN IT...*sigh*

File: README.md [hunk 5]
```diff
@@ -1,17 +1,36 @@
...
+  2. cd to project root
+  3. `npm install -g gulp-cli`
+  4. `npm install`
+
+### Deployment
+There is no stable deployment version yet.
+There is no demo or example set up yet.
+
+## Bug / Feature Request
+If you find a bug, kindly open a
```
Comment (Gander7) [inline README.md]: That was me, jerk :P
Shoot I broke our contribution terms

File: README.md [hunk 6]
```diff
@@ -19,17 +19,14 @@ Web application project to create a real time prayer wall so users can submit pr
...
-
 ## Contributing
 Please read [CONTRIBUTING.md](../CONTRIBUTE.md) for details on the process of contributing a bug report or a new feature request.
 
 ## Versioning
 We use [SemVer](http://semver.org/) for versioning.
 
 ## Owners
-- Jamal Rahali
+- Jamal Rahhali
```
Comment (plusjamal) [inline README.md]: It's actually Rahhhali.  Three h's. lol, JK!

**Comment by Gander7** [comment]
Good starting point, discussion to follow regarding next step.

**Pull request merged**